### PR TITLE
Update infection model to default status to active

### PR DESCRIPTION
**Pull request by KseniiaNovinskaia**

### CODE REVIEW

File: app/controllers/infections_controller.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def create
     @infection = Infection.new(infection_params)
     @infection.organism = @organism
     @infection.user = current_user
+    @infection.status = "active"
```
Comment (Kimon-Haars) [inline app/controllers/infections_controller.rb]: do we really need this, if the column already has a default value?

Comment (KseniiaNovinskaia) [reply app/controllers/infections_controller.rb]: @Kimon-Haars you are right, it was left from my initial testing. let me remove it.

**Comment by KseniiaNovinskaia** [comment]
@Kimon-Haars removed it.

**Comment by Kimon-Haars** [comment]
looks great. 🎉

**Comment by sabinacuccibar** [comment]
to me looks all good here!

**Review by sabinacuccibar — APPROVED** [review]

**Review by Kimon-Haars — APPROVED** [review]

**Pull request merged**